irods_specific_implementation/irods_rest_stream_get_api_implementation.h

```diff
@@ -3,11 +3,11 @@
 
 #include "irods_rest_api_base.h"
 
-#include "dstream.hpp"
-#include "transport/default_transport.hpp"
-#include "rodsErrorTable.h"
-#include "irods_exception.hpp"
-#include "ticketAdmin.h"
+#include <irods/dstream.hpp>
+#include <irods/transport/default_transport.hpp>
+#include <irods/rodsErrorTable.h>
+#include <irods/irods_exception.hpp>
+#include <irods/ticketAdmin.h>
 
 #include "pistache/http.h"
 #include "pistache/optional.h"
```

Comment:
We ship the pistache library as an externals package.

You might want to consider changing these headers to use angle brackets too.

Reply:
I plan on doing a consistency sweep for 5712 separately. I'm keeping the 6201 companion PRs small so we can get it merged sooner rather than later.